reconfig.js

```diff
@@ -94,8 +94,11 @@ Reconfig.prototype.resolveReferences = function(value) {
  * @return {*}
  */
 Reconfig.prototype.resolveParameters = function(value, parameters) {
+  var paramSrc = '';
+
   for (var property in parameters) {
-    value = (value) ? value.replace(':' + property, (parameters[property] || '')) : value;
+    paramSrc = RegExp(':' + property, 'g');
+    value = (value) ? value.replace(paramSrc, (parameters[property] || '')) : value;
```

Comment:
why not avoiding creating the `paramSrc` to begin with?

``` javascript
value = (value) ? value.replace(RegExp(':' + property, 'g'), (parameters[property] || '')) : value;
```

BTW instead of the ternary operatoor I think it might be more readable to do something like:

``` javascript
if (value) {
  value = value.replace(paramSrc, (parameters[property] || ''));
}
```